hack/prepare-release.sh

```diff
@@ -14,8 +14,8 @@ VERSION=${released_version} IMAGE_TAG=${prefixed_released_version} make gen-mani
 git add manifests/cluster-network-addons/${released_version}
 
 echo 'Upgrade README.md with the released manifests'
-sed -i "s/\(.*kubectl apply.*\)${prefixed_previous_version}\(.*\)/\1${prefixed_released_version}\2/g" README.md
-sed -i "s/\(.*startingCSV.*\)${prefixed_previous_version}\(.*\)/\1${prefixed_released_version}\2/g" README.md
+sed -i "s/\(.*kubectl apply.*\)${previous_version}\(.*\)/\1${released_version}\2/g" README.md
```

Comment:
if we're using the release version link (example: https://github.com/kubevirt/cluster-network-addons-operator/releases/download/v0.42.2/namespace.yaml) 
then I think the `v` **is** needed, right?
if so then this specific commit is not needed.

Reply:
> then I think the v is needed, right?

You're correct, of course. I'm leaving the prepare-release.sh as it was and just changing the link in README.
Also, after testing `make prepare-minor` properly, I'm using `v0.42.0` version as that's what we have in https://github.com/kubevirt/cluster-network-addons-operator/blob/043348fec507cab45501c5d859b9fd76ddac5ecc/version/version.go#L4